homeassistant/components/assist_pipeline/pipeline.py

```diff
@@ -1205,7 +1221,7 @@ def _create_item(self, item_id: str, data: dict) -> Pipeline:
 
     def _deserialize_item(self, data: dict) -> Pipeline:
         """Create an item from its serialized representation."""
-        return Pipeline(**data)
+        return Pipeline.from_json(data)
```

Comment:
Add a comment when this was introduced, so we know that older versions will still break.

Reply:
Comment added to `Pipeline.from_json`